awx/ui/src/screens/Job/JobOutput/useJobEvents.js

```diff
@@ -42,6 +47,31 @@ export default function useJobEvents(callbacks, isFlatMode) {
     });
   }, [actionQueue]);
 
+  useEffect(() => {
+    if (isFlatMode) {
+      return;
+    }
+
+    callbacks
+      .fetchChildrenSummary()
```

Comment:
good idea. I chatted with @akus062381 and we'll look to clean this up in a follow-up PR to avoid changes this late in testing. I can delete the unused JobEventsAPI at the same time

Reply:
#12011 filed